docs/book/src/topics/eks/pod-networking.md

```diff
@@ -7,8 +7,26 @@ When creating a EKS cluster the Amazon VPC CNI will be used by default for Pod N
 ## Using the VPC CNI Addon
 You can use an explicit version of the Amazon VPC CNI by using the **vpc-cni** EKS addon. See the [addons](./addons.md) documentation for further details of how to use addons.
 
+## Using Custom VPC CNI Configuration
+If your use case demands [custom networking](https://docs.aws.amazon.com/eks/latest/userguide/cni-custom-network.html) VPC CNI configuration you might already be familiar with the [helm chart](https://github.com/aws/amazon-vpc-cni-k8s) which helps with the process. This gives you access to ENI Configs and you can set Environment Variables on the `aws-node` daemonset where the VPC CNI runs. The Cluster API Provider is able to tune the same daemonset through the API.
 
-## Increase node pod limit
+The following example shows how to turn on custom network config and set a [label definition](https://github.com/aws/amazon-vpc-cni-k8s#eni_config_label_def).
```

Comment:
> [label definition]

Am I missing something here? These are environment properties that we are setting not labels. Right? :D

Reply:
label definition is what they call that setting it's what they use to look up nodes by label and do whatever it is they do. that's why i linked to the helm chart docs to show people what im referencing.

Reply:
Very weird stuff.

Reply:
networkin' aint easy